code/modules/clothing/under/clerksuits.dm

```diff
@@ -122,7 +122,7 @@
 
 /obj/item/clothing/under/suit/lobotomy/kcorp
 	name = "k-corp officer uniform"
-	desc = "A plain black suit with green tie. Used by K-corp non-combat staff."
```

Comment:
This is accurate I think, we have other outfits for K-Corp they're just the old Centcom outfits.

Reply:
Ah cool, i'll snag them then.